microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md

```diff
@@ -48,39 +46,26 @@ The labs are also designed to be connected to trials for:
 
 ## Step-by-step labs
 
-Detailed lab guides take you through multiple deployment and management scenarios. The labs have been updated for the latest versions of Intune and Configuration Manager.
-
-Note: The Windows 11 version of the lab includes the Windows 10 lab guide. A Windows 11 version of the lab guide will be available soon. 
+Detailed lab guides take you through multiple deployment and management scenarios. The labs have been updated for the latest versions of Intune and Configuration Manager. Note: A new Windows 11 version of the lab is now available. The lab guides include the following scenarios: 
 
 ### Plan and prepare infrastructure
 
 - Desktop Analytics
```

Comment:
You removed DA from the overview paragraph at the top, do you still want to include it here?